### PR TITLE
Fix SSL renew hook & simplify install cert command

### PR DESCRIPTION
**Pull request by ArthurPatriot**

### CODE REVIEW

File: docs/Maintain/build-and-start-a-node/03-ssl-certs-for-node.md [hunk 1]
```diff
@@ -15,17 +15,22 @@ If you need an SSL certificate for your node, follow the steps below:
 
    ```bash
    acme.sh --issue --standalone -d your_node.example.com \
-   --renew-hook "cd /opt/your_node; ./stop.sh; ./start.sh"
+   --renew-hook "docker restart tpnode" <!-- Use this if you run with docker -->
+   --renew-hook "./bin/thepower stop && ./bin/thepower foreground" <!-- Use this if you run from source code -->
    ```
+   
+   :::info
```
Comment (EZabotkin) [inline docs/Maintain/build-and-start-a-node/03-ssl-certs-for-node.md:22]: Add the empty lines
:::info

Use only one `--renew-hook`, depending on the run option (from docker / from source code)

:::

**Review by EZabotkin — CHANGES_REQUESTED** [review]

File: docs/Maintain/build-and-start-a-node/03-ssl-certs-for-node.md (hunk 1 above)
Comment (EZabotkin) [reply docs/Maintain/build-and-start-a-node/03-ssl-certs-for-node.md:22]: Please, make some minor changes

**Comment by ArthurPatriot** [comment]
Added fix for allowing run docker from not node folder. @giDai7ja

File: docs/Maintain/build-and-start-a-node/05-startingTpNode_docker.md [hunk 2]
```diff
@@ -48,10 +48,10 @@ To start the node run the following command:
 docker run -d \
 --name tpnode \
 --restart unless-stopped \
---mount type=bind,source="$(pwd)"/db,target=/opt/thepower/db \
---mount type=bind,source="$(pwd)"/log,target=/opt/thepower/log \
---mount type=bind,source="$(pwd)"/node.config,target=/opt/thepower/node.config \
---mount type=bind,source="$(pwd)"/genesis.txt,target=/opt/thepower/genesis.txt \
+--mount type=bind,source=/opt/thepower/db,target=/opt/thepower/db \
```
Comment (giDai7ja) [inline docs/Maintain/build-and-start-a-node/05-startingTpNode_docker.md:51]: and this can be done, not fundamentally

**Review by giDai7ja — APPROVED** [review]
it's ok

**Review by EZabotkin — APPROVED** [review]

**Pull request merged**